jaeger-core/src/main/java/io/jaegertracing/spi/Sender.java

```diff
@@ -12,10 +12,10 @@
  * the License.
  */
 
-package io.jaegertracing.senders;
+package io.jaegertracing.spi;
 
-import io.jaegertracing.Span;
-import io.jaegertracing.exceptions.SenderException;
+import io.jaegertracing.internal.Span;
```

Comment:
Another internal class passed to SPI.

Reply:
Perhaps the `Sender` should accept an OpenTracing Span?